topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,213 @@
+
+    5.  `plots :`
+        A plot is generated for each significant association.
+        Scatter plots are used for continuous metadata.
+        Box plots are for categorical data.
+        Data points plotted are after normalization, filtering, and transform.
+
+   > <question-title></question-title>
+    >
+    > 1. Open the heatmap.pdf output file and observe the heatmap. What do you notice?
+    > 2. What do you observe in the significant.tsv file?
+    >
+    > > <solution-title></solution-title>
+    > > 1. You will observe the association of the microbes with the fixed effects variables as `age` and `diagnosis`.
+    > > - Observe how setting the reference value as `CD` for the categorical variable `diagnosis` in MaAsLin2 implies that this reference level will be used as the baseline for comparison against other levels of the variable, i.e, `nonIBD` and `UC`.
+    > > - The effects of other levels will be interpreted relative to this reference level, helping to understand their impact on microbial features.
+    > > - The **colors** of the heatmap represent the magnitude and direction of associations between microbial features and metadata variables.
+    >       > - **Color Intensity** : The intensity of the color indicates the strength of the association. Darker or more vivid colors usually represent stronger associations.
+    >       > - **Color Hue** : The hue (e.g., red, blue) typically indicates the direction of the association. For instance, red represents positive associations (where an increase in the metadata variable is associated with an increase in the microbial feature) and another color blue represents negative associations (where an increase in the metadata variable is associated with a decrease in the microbial feature).\
+    >       >
+    >       > For example, if you look for `Bifidobacterium longum` in the heatmap, you'll notice that its occurrence in the human gut is least affected by the individual's age and shows a neutral effect in relation to their diagnosis of UC (Ulcerative Colitis) and non-IBD (non-Inflammatory Bowel Disease).
+    > > 2. The significant.tsv file shows statistically significant associations between microbial features and metadata variables that meet a specified threshold (in our case, the default `Maximum significance = 0.25`). It includes effect sizes, p-values, and adjusted p-values (q-values) to indicate the strength, direction, and reliability of each association. This file helps identify meaningful relationships in the microbiome data.
+    > {: .solution }
+    {: .question}
+
+
+
+
+
+# Conclusion
+
+In essence, uncovering associations between microbial features and metadata variables through tools like MaAsLin2 not only deepens our understanding of microbiome dynamics but also holds promise for clinical applications, personalized health strategies, and advancing the field of microbiome research.
```

Comment:
Maybe explain how these findings can be further visualized using tools like ampvis2 and phyloseq ...